### DISCUSSION TITLE
EXPORT_FILE_NAME for Multilingual Hugo Site does not work like expected

### DISCUSSION THREAD

**Discussion by freylax** [kaushalmodi/ox-hugo#661] [Q&A] [answered]
When creating a multilingual HUGO Website one  possible file naming has the form
`foo.en.md` and `foo.de.md`  for having an english and german page. This form has the
benefit that shared content can be local like the bundle philosophy. The other way is 
to use different directories for each language. If we will stick to the first form we get a 
problem if we want to use subtree based hugo exports, if specifying for eg
`EXPORT_FILE_NAME: foo.en`
this will export to a file `foo.md` and not to `foo.en.md`
It seems that the extension `.en` is just ignored.

**Comment by kaushalmodi** [comment, marked as answer]
This should work

```
:EXPORT_FILE_NAME: foo.en.md
```

**Reply by freylax** [reply]
Thank you @kaushalmodi , this works as expected!
